src/test/e2e/swagger/endpoints.spec.ts

```
@@ -0,0 +1,154 @@
+  test('should display main Swagger UI components', async () => {
+    await expect(elements.apiDocumentation).toBeVisible();
+    await expect(elements.navigation).toBeVisible();
+    await expect(elements.authorizeButton).toBeVisible();
+    await expect(elements.schemeContainer).toBeVisible();
+  });
+
+  test('should display API endpoints', async () => {
+    await elements.endpoints.first().waitFor({ state: 'visible' });
+    const endpointsCount: number = await elements.endpoints.count();
+
+    for (let i: number = 0; i < endpointsCount; i += 1) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)